tikv-client/src/main/java/com/pingcap/tikv/expression/ArithmeticBinaryExpression.java

```diff
@@ -42,6 +42,11 @@ public static ArithmeticBinaryExpression minus(Expression left, Expression right
     return new ArithmeticBinaryExpression(left.dataType, MINUS, left, right);
   }
 
+  public static ArithmeticBinaryExpression multiply(
```

Comment:
what about other arithmetic binary expressions, do them need add similar function?

Reply:
For now, we only find multiply and division need this.